webapp/channels/src/i18n/en.json

```diff
@@ -4438,16 +4438,16 @@
   "notification.crt": "Reply in {title}",
   "notification.dm": "Direct Message",
   "notify_all.confirm": "Confirm",
-  "notify_all.question": "By using **@all** or **@channel** you are about to send notifications to **{totalMembers} people**. Are you sure you want to do this?",
-  "notify_all.question_groups": "By using **{mentions}** and **{finalMention}** you are about to send notifications to at least **{totalMembers} people**. Are you sure you want to do this?",
-  "notify_all.question_one_group": "By using **{mention}** you are about to send notifications to **{totalMembers} people**. Are you sure you want to do this?",
-  "notify_all.question_timezone": "By using **@all** or **@channel** you are about to send notifications to **{totalMembers} people** in **{timezones, number} {timezones, plural, one {timezone} other {timezones}}**. Are you sure you want to do this?",
-  "notify_all.question_timezone_groups": "By using **{mentions}** and **{finalMention}** you are about to send notifications to at least **{totalMembers} people** in **{timezones, number} {timezones, plural, one {timezone} other {timezones}}**. Are you sure you want to do this?",
-  "notify_all.question_timezone_one_group": "By using **{mention}** you are about to send notifications to **{totalMembers} people** in **{timezones, number} {timezones, plural, one {timezone} other {timezones}}**. Are you sure you want to do this?",
   "notify_all.title.confirm": "Confirm Sending Notifications to Entire Channel",
   "notify_all.title.confirm_groups": "Confirm sending notifications to groups",
-  "notify_here.question": "By using **@here** you are about to send notifications to up to **{totalMembers} other people**. Are you sure you want to do this?",
```

Comment:
@hmhealey do we want to set the standard to camelCase instead of snake_case? I have been seeing this kind of changes and I am unsure what should be the correct standard?

Reply:
This is scheduled for web guild next week. We can decide then and not in this PR